ui/admin/app/components/form/field/list-wrapper/index.js

```diff
@@ -16,6 +16,8 @@ export default class MappingListComponent extends Component {
    */
   @tracked newOptionKey = '';
 
+  @tracked newRow = '';
```

Comment:
None blocking comment: I do understand newRow as a temporary variable to persist the values of the new row before "adding" the row (addRow) is that correct?

Reply:
This isn't needed anymore, but you are right. That was my original intention.